components/gitpod-db/src/typeorm/project-db-impl.ts

```diff
@@ -62,7 +62,10 @@ export class ProjectDBImpl extends TransactionalDBImpl<ProjectDB> implements Pro
 
     public async findProjectsByCloneUrl(cloneUrl: string, organizationId?: string): Promise<Project[]> {
         const repo = await this.getRepo();
-        const conditions: FindConditions<DBProject> = { cloneUrl, markedDeleted: false, teamId: organizationId };
```

Comment:
:orange_heart: for adding a test for the function [here](https://github.com/gitpod-io/gitpod/blob/eaae6cc6f4f5ee1ec81db4c611502f5c75fca97d/components/gitpod-db/src/project-db.spec.db.ts#L169)!

Reply:
Working on it! 👨‍💻

Reply:
✅